package.json

```diff
@@ -57,7 +57,6 @@
 		"@typescript-eslint/eslint-plugin": "^4.23.0",
 		"@typescript-eslint/parser": "^4.23.0",
 		"eslint": "^7.26.0",
-		"tsc-watch": "^4.2.9",
-		"typescript": "^4.1.3"

```

Comment:
I'd say put the typescript version number back (just to make sure a later version won't mess with anything). TS is on 4.6.2 as of right now, so I don't want anything to accidentally break